charts/lib/values.yaml

```diff
@@ -0,0 +1,10 @@
+exports:
+  global:
+    lib:
+      persistentVolume:
+        storageCapacity: '750Gi'
+        accessMode: ReadWriteOnce
```

Comment:
Is this the correct access mode for local installs? If this is where all data goes including postgres I do not think that will work

Reply:
Yes... the `ReadWriteOnce` means it's loaded on one `node` but can be loaded read-write by multiple pods.

Reply:
For more context, the alternative `ReadWriteMany` is, I believe, not supported for `local` volumes and is more meant for something like `NFS`.